superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml

```diff
@@ -64,7 +64,7 @@ superset_postgres_db_user: "{{ superset_sql_conn_creds['admin']['postgres']['use
 superset_postgres_db_password: "{{ superset_sql_conn_creds['admin']['postgres']['password'] }}"
 
 
-superset_host_port: "8888"
```

Comment:
Now, we are accessing superset through nginx. So you need to use nginx port "superset_nginx_port" variable. We are using **superset_host_port** internally by nginx to access superset

Reply:
done